dapp/src/utils/networks.ts

```diff
@@ -0,0 +1,44 @@
+export const networks = {
+  testnet: {
+    name: "Testnet",
+    passphrase: "Test SDF Network ; September 2015",
+    rpc: "https://soroban-testnet.stellar.org:443",
+    horizon: "https://horizon-testnet.stellar.org",
+    tansu_contract: import.meta.env.PUBLIC_TANSU_CONTRACT_ID,
+    domain_contract: import.meta.env.PUBLIC_SOROBAN_DOMAIN_CONTRACT_ID,
+  },
+  mainnet: {
+    name: "Mainnet",
+    passphrase: "Public Global Stellar Network ; September 2015",
+    rpc: "https://rpc.lightsail.network",
+    horizon: "https://horizon.stellar.org",
+    tansu_contract: import.meta.env.PUBLIC_TANSU_CONTRACT_ID,
+    domain_contract: import.meta.env.PUBLIC_SOROBAN_DOMAIN_CONTRACT_ID,
```

Comment:
We need to make new env variables for these otherwise that does not make sense. Contracts have diff addresses on mainnet and testnet.